src/lib/Preview.js

```diff
@@ -745,6 +767,9 @@ class Preview extends EventEmitter {
             this.throttledMousemoveHandler
         );
 
+        // Set up the notification
+        this.ui.setupNotification();
```

Comment:
Please work with @pramodsum to make sure this doesn't muck with annotations. I remember we've had a few issues with the location of `setupNotification()` before.

Reply:
Were you able to resolve this @jeremypress @pramodsum?